adapters/slog/slog.go

```diff
@@ -69,6 +70,14 @@ func SetLevel(level slog.Leveler) Option {
 	}
 }
 
+// SetAddSource specifies whether to add the source of the log message to The Event.
+func SetAddSource(addSource bool) Option {
+	return func(h *Handler) error {
+		h.addSource = addSource
+		return nil
+	}
+}
```

Comment:
I don't have an extremely strong opinion here and I also believe it comes down to personal preference, **but** to be [consistent with existing options that set a boolean](https://github.com/axiomhq/axiom-go/tree/main/axiom/client_options.go#L78-L94), I suggest to not take a parameter and just set `true`:

```suggestion
func SetAddSource() Option {
	return func(h *Handler) error {
		h.addSource = true
		return nil
	}
}
```

WDYT?

Reply:
My opinion on this is nil.
i just coded this method really quick without paying much attention for the others options that set a bool.